src/modules/xcm/tokens/index.ts

```diff
@@ -177,6 +177,16 @@ export const xcmToken = {
       originChain: Chain.KINTSUGI,
       minBridgeAmount: '0.345',
     },
+    {
+      symbol: 'CSM',
+      isNativeToken: true,
+      assetId: '18446744073709551624',
+      originAssetId: 'CSM',
+      logo: require('/src/assets/img/shadow.svg'),
+      isXcmCompatible: true,
+      originChain: Chain.CRUST_SHADOW,
+      minBridgeAmount: '0.001',
```

Comment:
The minimum bridged amount is calculated to be 118.547 CSM based on our `unitsPerSecond`